aws/aws.go

```diff
@@ -69,3 +78,24 @@ func WriteToShell(c *Credentials, windows bool, w io.Writer) {
 		)
 	}
 }
+
+// GetNonExpiredCredentials Return profiles which have a aws_expiration key but are not yet expired
```

Comment:
Please add a period at the end :-)

This function could return profiles which weren't created by Clisso... I'm thinking whether we should prefix all profiles Clisso saves with `clisso-`. This would also make it clearer we're using temporary credentials when doing `export AWS_PROFILE=profile`. WDYT?